frontend/src/components/_common/Buttons/Button/index.tsx

```diff
@@ -20,13 +22,14 @@ export function Button({
   disabled,
   type = 'button',
   onClick,
+  customCss,
 }: ButtonProps) {
   const cssProps = [s_baseButton(borderRadius), s_size(size)];
 
   if (variant) cssProps.push(s_variant[variant]);
 
   return (
-    <button disabled={disabled} css={cssProps} type={type} onClick={onClick}>
+    <button disabled={disabled} css={[cssProps, customCss]} type={type} onClick={onClick}>
```

Comment:
[P3]

`<button disabled={disabled} css={[[s_bseButton(borderRadius), s_size(size)], customCss]} type={type} onClick={onClick}>` 와 같은 형태로 된 것 같은데 아래처럼 바꾸어야 되지 않을까용? 실제로 잘 동작 하던가요?!?!?

```suggestion
    <button disabled={disabled} css={[...cssProps, customCss]} type={type} onClick={onClick}>
```

Reply:
잘 동작하던데용???